packages/atomic/src/components/ipx/atomic-ipx-embedded/atomic-ipx-embedded.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Component,
+  h,
+  State,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+} from '@stencil/core';
+import {
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {updateBreakpoints} from '../../../utils/replace-breakpoint';
+import {once, randomID} from '../../../utils/utils';
+import {AnyBindings} from '../../common/interface/bindings';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-ipx-embedded',
+  styleUrl: 'atomic-ipx-embedded.pcss',
+  shadow: true,
+})
+export class AtomicIPXEmbedded implements InitializableComponent<AnyBindings> {
+  @InitializeBindings() public bindings!: AnyBindings;
+  @Element() public host!: HTMLElement;
+
+  @State() public error!: Error;
+
+  /**
+   * The container to hide from the tabindex and accessibility DOM when the modal is closed.
+   */
+  @Prop({mutable: true}) container?: HTMLElement;
```

Comment:
You should sync with https://github.com/coveo/ui-kit/pull/2697

Reply:
Looks like it's a different issue, there's actually more to making `atomic-ipx-embedded` than removing the focus trap, especially looking at the css of both variants.
We're gonna need their feature for the refine modal though, since we don't want there to be a focus trap, as the page is embedded, I guess we'll wait for that to be merged and sync !

Reply:
> We're gonna need their feature for the refine modal though, since we don't want there to be a focus trap, as the page is embedded, I guess we'll wait for that to be merged and sync !

Concerning the option to remove the focus trap, this is actually very similar to the context for the refine case. They don't want elements behind the modal to be focusable, but they also don't want to prevent users from tabbing out of the search page itself.

Either way, their fix should work for IPX too.